docroot/modules/humsci/hs_blocks/src/Plugin/Block/GroupBlock.php

```diff
@@ -180,25 +192,36 @@ protected function buildAdministrativeSection(array &$components) {
       ];
     }
 
+    $route_params = [
+      'section_storage_type' => $section_storage->getStorageType(),
+      'section_storage' => $section_storage->getStorageId(),
+      'delta' => $section_delta,
+      'group' => $this->configuration['machine_name'],
+    ];
+
+    // Build a contextual id that won't generate errors from ajax.
+    // @see _contextual_id_to_links()
+    // Use contextual link data attributes so that layout builder doesn't
+    // disable the link from being clicked.
+    // @see behaviors.layoutBuilderDisableInteractiveElements() in layout-builder.js
+    $contextual_id = $this->configuration['machine_name'] . ':' . http_build_query($route_params) . ':langcode=en';
```

Comment:
I know we don't do multilang but wouldn't it be better to grab the active language instead of hardcoding it here?

Reply:
it doesnt really do anything in the end. i just grabbed what another contextual link has. it could probably even be an empty string. just need 2 of the `:` in there for the `list` and `explode` in contextual module